README.md

```diff
@@ -61,7 +61,7 @@ You can download .NET Core SDK as either an installer (MSI, PKG) or a zip (zip,
 In order to download just the .NET Core runtime without the SDK, please visit https://github.com/dotnet/core-setup#daily-builds.
 
 > **Note:** please be aware that below installers are the **latest bits**. If you 
-> want to install the latest released versions, please check out the [section above](#download-links).)
+> want to install the latest released versions, please check out the [section above](#looking-for-v1-of-the-net-core-tooling).
```

Comment:
That was intentional, there is no opening parenthesis, so there shouldn't be a closing one either.

I do see the table broken [here](https://github.com/svick/cli/blob/7ebee29dc1a231dee0e53fc44a6cc23112e945b6/README.md), but [it looks broken in the previous version too](https://github.com/svick/cli/blob/09b6be599007ea4d791dc4b7bbfa63541e819419/README.md) (which looks fine on the main page of the repo), so I don't think it's related to the parenthesis.